src/cache/lru_cache_manager/lru_cache_manager.h

```diff
@@ -49,7 +49,15 @@ CacheData *evictFromCache(LRUCacheManager *lruCacheManager);
  * @param  *cacheData: Cache entry address (pointer)
  * @retval None
  */
-void increaseImportance(LRUCacheManager *cacheManager, void *cacheData);
+void increaseImportance(LRUCacheManager *cacheManager, CacheData *cacheData);
```

Comment:
To me, it makes more sense to have increaseImportance an internal function of cache.c

Reply:
file removed